README.md

```diff
@@ -0,0 +1,83 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# FPS Games Global CON 2022
+
+> The global event of first person shooter games
+
+Microverse Capstone Project - HTML/CSS & JavaScript Conference page
+
+
+## Live Server
+
+In the following link, you can view [FPSCON 2022](https://pepedropo1.github.io/Portfolio/).
```

Comment:
- [ ] I think you forgot to change the link of website because it is opening your portfolio project. Please kindly change this link to your Conference website live demo link 😊👍